src/modules/Dropdown/Dropdown.js

```diff
@@ -425,64 +422,57 @@ export default class Dropdown extends Component {
         if (openOnFocus) this.open()
       }
       if (!this.state.open) {
-        document.addEventListener('keydown', this.openOnArrow)
-        document.addEventListener('keydown', this.openOnSpace)
-      } else {
-        document.addEventListener('keydown', this.moveSelectionOnKeyDown)
-        document.addEventListener('keydown', this.selectItemOnEnter)
+        eventPool.sub('keydown', [this.openOnArrow, this.openOnSpace])
       }
-      document.addEventListener('keydown', this.removeItemOnBackspace)
```

Comment:
`moveSelectionOnKeyDown`, `selectItemOnEnter` and `removeItemOnBackspace` should be added only if the `Dropdown` is open